packages/workers/src/runner.ts

```diff
@@ -26,19 +31,9 @@ export async function runner(dbUrl: string, chainId: EvmChain, logger: Logger) {
     logger,
   })
 
-  // TODO: Remove this once we have support for BSC and Fantom
-  if (chainId === Chain.Bsc || chainId === Chain.Fantom) {
```

Comment:
Removed this from here. It'll use the library defined providers to determine if the chain is enabled or not, the same way the API already does it.